CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Change Log - Salesforce Code Analyzer VS Code Extension (Beta)
+
+To see a list of changes each month for the Salesforce Code Analyzer VS Code Extension (Beta), visit our [Salesforce Code Analyzer Release Notes](https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/release-notes.md) page.
```

Comment:
@teresa-allen-sfdc - I think it'll be good for you review this. This eventually goes into the changelog section of the extension marketplace - https://marketplace.visualstudio.com/items/salesforce.sfdx-code-analyzer-vscode/changelog.

Reply:
Sorry, didn't see your comment until after I already merged (via Josh's approval). I just pinged @teresa-allen-sfdc internally.